superset-frontend/src/dashboard/components/nativeFilters/FiltersConfigModal/FiltersConfigForm/FiltersConfigForm.tsx

```diff
@@ -104,17 +104,21 @@ export const FiltersConfigForm: React.FC<FiltersConfigFormProps> = ({
   // @ts-ignore
   const hasDatasource = !!nativeFilterItems[formFilter?.filterType]?.value
     ?.datasourceCount;
+  const hasColumn =
+    hasDatasource &&
+    !['filter_timegrain', 'filter_timecolumn'].includes(formFilter?.filterType);
```

Comment:
1. I think need move it to `consts`
2. May be we can do it a little bit common, like per some metadata value?

Reply:
Good idea, I'll move it to to constants. This is still pending a discussion around how much in common filter configs should have with the explore control panel, but for now a constant lookup should be ok.